### PR TITLE
[Messenger] Fixed ConsumeMessagesCommand configuration

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ protected function configure(): void
                 new InputOption('memory-limit', 'm', InputOption::VALUE_REQUIRED, 'The memory limit the worker can consume'),
                 new InputOption('time-limit', 't', InputOption::VALUE_REQUIRED, 'The time limit in seconds the worker can run'),
                 new InputOption('sleep', null, InputOption::VALUE_REQUIRED, 'Seconds to sleep before asking for new messages after no messages were found', 1),
-                new InputOption('bus', 'b', InputOption::VALUE_REQUIRED, 'Name of the bus to which received messages should be dispatched (if not passed, bus is determined automatically.'),
+                new InputOption('bus', 'b', InputOption::VALUE_REQUIRED, 'Name of the bus to which received messages should be dispatched (if not passed, bus is determined automatically).'),
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php]: the dot should be before the bracket :)

Comment (lyrixx) [reply src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php]: Let's remove the dot?

Comment (chalasr) [reply src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php]: 👍 for removing it

Comment (lyrixx) [reply src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php]: Fixed

**Review by chalasr — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Good catch, thanks @lyrixx.

**Pull request merged**